packages/react-components/react-combobox/src/components/Combobox/useComboboxStyles.ts

```diff
@@ -1,30 +1,166 @@
+import { tokens } from '@fluentui/react-theme';
 import { SlotClassNames } from '@fluentui/react-utilities';
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import { iconSizes } from '../../utils/internalTokens';
 import type { ComboboxSlots, ComboboxState } from './Combobox.types';
 
 export const comboboxClassNames: SlotClassNames<ComboboxSlots> = {
   root: 'fui-Combobox',
+  button: 'fui-Combobox__button',
+  expandIcon: 'fui-Combobox__expandIcon',
   listbox: 'fui-Combobox__listbox',
-  trigger: 'fui-Combobox__trigger',
 };
 
 /**
- * Styles for the root slot
+ * Styles for Combobox
  */
 const useStyles = makeStyles({
-  root: {},
+  root: {
```

Comment:
The `root` slot is a `div` which has a default `display` value of `block` but all our input controls have a default display value of `inline` (or `inline-block`, `inline-flex`, etc).

Should this slot have `display: inline-block` set to make it consistent with other input controls?